app/models/service.rb

```diff
@@ -146,6 +146,16 @@ class Service < ActiveRecord::Base
         resource.sites.map(&:site_code)
       end
 
+      add_attribute :documents do
+        documents.map do |d|
+          { id: d.id, url: d.url, description: d.description }
```

Comment:
The id is included so that the React frontend can use it to maintain a map between each specific document object and UI changes